### PR TITLE
fix(library): wdi5 assertions without browser service

### PR DESCRIPTION
**Pull request by QA-ler**

### CODE REVIEW

**Comment by QA-ler** [comment]
If chromedriver service is not used the domElement is not set accordingly via devtool protocol.
Therefore we get the element reference by calling browser execute function manually

File: src/lib/wdi5-control.ts [hunk 1]
```diff
@@ -659,6 +659,23 @@ export class WDI5Control {
         }
 
         const _result = (await clientSide_getControl(controlSelector, this._browserInstance)) as clientSide_ui5Response
+
+        // When chromedriver service is not used the domElement is not set accordingly via devtool protocol
+        // Therefore we get element reference by calling browser execute function manually
+        if (_result.status === 0 && !_result.domElement[ELEMENT_KEY]) {
+            const elementReference = (await this._browserInstance.execute((id) => {
+                const webElement: Node = document.evaluate(
```
Comment (vobu) [inline src/lib/wdi5-control.ts:667]: black magic here :)
I would tend to see this async, meaning `await this._browserInstance.executeAsync` but I'm ok with this in here b/c this part is executing client-side already is in a "needs to run" in order thing

**Review by vobu — COMMENTED** [review]
again, nice finding!
I'll merge PR #404 so that these 2 can get in together

File: src/lib/wdi5-control.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import * as util from "util"
-
+import { ELEMENT_KEY as _ELEMENT_KEY } from "webdriverio/build/constants"
```
[the remark this answers is not in the document]
Comment (vobu) [reply src/lib/wdi5-control.ts]: i actually changed this myself :)

File: src/lib/wdi5-control.ts (hunk 1 above)
Comment (vobu) [reply src/lib/wdi5-control.ts:666]: i take this back - for some reason, `await`ing`this._browserInstance.execute` is necessary - guessing an issue with the event loop tick.
anyway, resolving this.

**Comment by vobu** [comment]
we're almost there.
why the auth tests fail in CI, but run fine locally, must be investigated - but this is not in scope of this PR.
so @QA-ler , if you'd please change the wording the comment as suggested above, we should be good to provide a first version of `wdi5` with devtools support!

**Comment by QA-ler** [comment]
@vobu Done

**Pull request merged**